packages/main/src/TableColumn.js

```diff
@@ -2,6 +2,7 @@ import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
 import litRender from "@ui5/webcomponents-base/dist/renderer/LitRenderer.js";
 import Integer from "@ui5/webcomponents-base/dist/types/Integer.js";
 import TableColumnTemplate from "./generated/templates/TableColumnTemplate.lit.js";
+import popinDisplay from "./types/TableCellPopinDisplayType.js";
```

Comment:
whatever the type is called, we use its name
`import TableCellPopinDisplayType from "./types/TableCellPopinDisplayType.js";`

- consider TableColumnPopinType as the "popinDisplay" property is being set on the column, the column pops in, so can't be too wrong, but it's shorter

Reply:
Thank you, I changed it.